example/shared/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ plugins {
     kotlin("multiplatform")
     kotlin("plugin.serialization")
     id("jacoco")
-    id("convention.publication")
+    id("maven-publish")
```

Comment:
Why do we have maven-publish in example?
It sounds as wrong as the one you removed. There shouldn't be any publishing in example